vtu11/inc/parallel_helper.hpp

```diff
@@ -0,0 +1,41 @@
+﻿//          __        ____ ____
+// ___  ___/  |_ __ _/_   /_   |
+// \  \/ /\   __\  |  \   ||   |
+//  \   /  |  | |  |  /   ||   |
+//   \_/   |__| |____/|___||___|
+//
+//  License: BSD License ; see LICENSE
+//
+#ifndef VTU11_PARALLEL_HELPER_HPP
+#define VTU11_PARALLEL_HELPER_HPP
+
+#include <array>
+#include "inc/alias.hpp"
+#include "vtu11.hpp"
+#include "inc/utilities.hpp"
+namespace vtu11
+{
+  //writes the .pvtu file, that keeps together all vtu pieces
+  template<typename Writer>
+  
+  void writePVTUfile( const std::string & path,
+                      std::string baseName,
+                      const std::vector<DataSet> & pointData,
+                      const std::vector<DataSet> & cellData,
+                      size_t numberOfFiles,
+                      Writer writer );
+
+namespace parallelHelper
+{
+  //This function adds an empty Parallel Dataset to the xml-format
+  template<typename Writer, typename DataType>
+  inline void addPEmptyDataSet( Writer& writer,
```

Comment:
why is this needed? Sounds counterintuitive to me

Reply:
We were also thinking about integrating this function into the already existing function "addDataSet()", which is used for writing the .vtu files.
But for using it to write the empty tags in the pvtu file, a little modification would be necessary. So we decided to better not modify your original function.
We were thinking about adding another boolean input parameter in addDataSets(), which tells us, if we want to write a tag in the pvtuFile or not. And then inside the function, we would add another if-else, where we write the PEmptyTag, if pvtuFile==true.
But I think it is a bit hard to describe our concept in here...
We will push a version in a separate branch of the two functions combined and you can have a look at the modified function and tell us, which way you prefer ;-)